index.html

```diff
@@ -49,180 +49,224 @@ <h2 class="hex connect">LET’S CONNECT</h2>
                 <ul>
-                <li class="lang-title">
-                <p class="langusge">Languages</p>
-                <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
-                </li>
-                <li class="lang">
-                <img src="images/langusges/js.png" alt="js" />
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/html.png" alt="js" />
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/css.png" alt="js" />
+                        <p>CSS</p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Frameworks</p>
+                        <p class="sign"></p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Skills</p>
+                        <p class="sign"></p>
+                    </li>
                 </ul>
-                </div>
             </div>
-
+        <section class="contact-me"> 
+            <div class="contact-background">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I'd love to help with
+                </p>           
+                <form action="https://formspree.io/f/{form_id}" method="post">
+                    <input type="text" id="fname" name="fname" placeholder="lorem Ipsum" maxlength="30" required><br>
+                    <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required><br>
+                    <textarea name="message" id="message" maxlength="500" required>Write your message here...</textarea><br>
```

Comment:
- [optional] please kindly use a white background for the text area so it follows the original design and makes the text more visible. 
yours
![Screenshot (940)](https://user-images.githubusercontent.com/65298048/230038391-4ff8aa7e-be59-4db0-af80-90119606a6b9.png)
figma
![Screenshot (939)](https://user-images.githubusercontent.com/65298048/230038485-98cf2de4-797c-4351-847e-6d18d705b6ff.png)